bokehjs/src/lib/models/dom/index.ts

```diff
@@ -1,17 +1,20 @@
-import {Model} from "../../model"
-import {LayoutDOM, LayoutDOMView} from "../layouts/layout_dom"
-import {Styles} from "./styles"
+import {build_views, remove_views} from "core/build_views"
 import {span} from "core/dom"
-import {View} from "core/view"
 import {DOMView} from "core/dom_view"
-import {build_views, remove_views} from "core/build_views"
 import * as p from "core/properties"
-import {isString} from "core/util/types"
+import {enumerate} from "core/util/iterator"
 import {entries} from "core/util/object"
-import * as styles from "styles/tooltips.css"
-
 import {Index as DataIndex, _get_column_value} from "core/util/templating"
+import {isString} from "core/util/types"
+import {View} from "core/view"
+import * as styles from "styles/tooltips.css"
+import {Model} from "../../model"
+import {LayoutDOM, LayoutDOMView} from "../layouts/layout_dom"
+/////
+import {RendererGroup} from "../renderers/renderer"
 import {ColumnarDataSource} from "../sources/columnar_data_source"
+import {Styles} from "./styles"
+
```

Comment:
Same here (regarding imports).

Reply:
removed the spurious comment line, not sure where it came from

Reply:
BTW is there a rationale for mixing "relative" and "absolute" imports? Is it required due to the way we split up the bundles? Or conventional due to that? Or just history?